lib/src/offline_region.dart

```diff
@@ -0,0 +1,70 @@
+part of mapbox_gl;
+
+class OfflineRegion {
+  const OfflineRegion({
+    @required this.id,
+    @required this.bounds,
+    @required this.metadata,
+    @required this.mapStyleUrl,
+    @required this.minZoom,
+    @required this.maxZoom,
+  });
+
+  final int id;
+  final LatLngBounds bounds;
+  final Map<String, dynamic> metadata;
+  final String mapStyleUrl;
+  final num minZoom;
```

Comment:
```suggestion
  final double minZoom;
```
small suggestion: zoom levels are always a double, this was throwing an 'int is not a subtype of double' error in the example/offline_region_map.dart.